security_group.tf

```diff
@@ -31,16 +31,22 @@ resource "aws_security_group_rule" "out_all" {
   cidr_blocks = ["0.0.0.0/0"]
 }
 
+locals {
+  not_ssh_open_ports = "${distinct(compact(
+    split(",", replace(join(",", var.in_open_ports), "/,22,|^22,|,22$|^22$|,22-22,|^22-22,|,22-22$|^22-22$/", ","))
```

Comment:
wow

Reply:
Yeah, I'm not really proud of this. As soon as we move provisioning the instance through Cloud-Init I'll get remove it.